### DISCUSSION TITLE
live display with `AboveNormal` and `Highest` priority threads

### DISCUSSION THREAD

**Discussion by mgood7123** [spectreconsole/spectre.console#822] [Q&A]
how can i get `Spectre.Console` to update a `Progress` on a `AboveNormal` or `Highest` priority thread?

**Comment by patriksvensson** [comment]
Moved to discussions.

**Comment by patriksvensson** [comment]
@mgood7123 The background thread for Progress is only updating things like state, and not for the actual work done. What is your use case for wanting to set the thread priority?

**Reply by mgood7123** [reply]
Monitoring progress of long running benchmarks without flooding the console with text

**Reply by mgood7123** [reply]
Specifically with the benchmark thread running at a high priority

**Reply by mgood7123** [reply]
specifically https://github.com/mgood7123/AndroidUI-Test-Framework/tree/main#benchmarking
